### ISSUE TITLE
Wrong date format (DD/MM -> MM/DD)

### ISSUE THREAD

**Issue by BuzzAntique** [Alamantus/GooglePhotosExportOrganizer#13]
Hi, I already found another solution here on GitHub (cmd tool), but there was an issue using this app (maybe due to my European region?). The date was fixed, but in the wrong format - the Date and Month were swapped (e.g. 12th of July (12/7) was changed to 7th of December (7/12)). Except for this issue great app!

**Comment by Alamantus** [comment]
@BuzzAntique Yeah, I'm based in America and of course didn't think of how the rest of the world writes their dates. 😅

Anyway, thanks for letting me know about this! I'll look into fixing this issue so the date parses correctly for day/month format instead of month/day. Unfortunately it may take me a little while to work on it because my current schedule doesn't allow me to work on my side projects much anymore, but once I have a solution, I'll tag you in an update!
